ddl/ddl_worker.go

```diff
@@ -188,7 +188,7 @@ func (d *ddl) handleDDLJobQueue() error {
 				return errors.Trace(err)
 			}
 
-			if job.IsRunning() {
+			if job.IsRunning() || job.IsSynced() {
```

Comment:
Why should we check if job is synced? if a job's state is synced, can we read it from ddl queue?